src/word_debt_bot/game/core.py

```diff
@@ -72,6 +72,16 @@ def get_player(self, player_id: str, optional=True) -> WordDebtPlayer | None:
         else:
             return self._state.users[player_id]
 
+    def get_player_by_name(
+        self, display_name: str, optional=True
+    ) -> WordDebtPlayer | None:
+        for user in self._state.users.values():
+            if user.display_name == display_name:
```

Comment:
At some point display names will be configurable by users, I'm not sure if we want to rely on this long-term when someone sets their display name to arbitrary unicode. A more extensive refactor that preserves the spirit is to add a separate `name` variable that is tied to the username and doesn't fall into unicode issues, but we could maybe also keep it as-is.

Reply:
The name in use right now in worddebt's display_name is the Discord username as I understand it. I'm don't really want to figure out how to migrate the journal into a version where it's called `name` instead right now, and I think the refactoring for this can fall under whatever PR handles adding display names